src/middleware/jsonld.ts

```diff
@@ -0,0 +1,20 @@
+import jsonld from 'jsonld';
+import { Context as JsonLdContext } from 'jsonld/jsonld-spec';
+import {
+  Context, Middleware, Next, Response,
+} from 'koa';
+
+const isJsonLd = (response: Response): boolean => response.type === 'application/ld+json' && typeof response.body === 'object';
+
+export default (context: JsonLdContext): Middleware => (
+  async ({ response }: Context, next: Next): Promise<void> => {
+    await next();
+
+    if (!isJsonLd(response)) {
+      return;
+    }
+
+    response.body = await jsonld.compact(response.body, context);
```

Comment:
is compacting of universal utility or should it be skipped when using the development target (like minification and similar)?

Reply:
I think it's useful generally. It will be replaced with profile negotation at some point too.